tutorials/time-series-visualization-with-altair/tutorial.ipynb

```diff
@@ -0,0 +1,2133 @@
+{
```

Comment:
<p>Suggest using an interactive chart, that highlights the selected year.</p><p> </p><p> </p><p><code># Chart NDVI time series as an Altair line chart.</code></p><p> </p><p><code>highlight = alt.selection(type='single', on='mouseover',</code></p><p><code>                          fields=['Year'], nearest=True)</code></p><p> </p><p><code>base = alt.Chart(ndvi_df).encode(</code></p><p><code>  alt.X('DOY:O'),</code></p><p><code>  alt.Y('NDVI:Q', scale=alt.Scale(domain=(0.1, 0.7))),</code></p><p><code>  alt.Color('Year:O', scale=alt.Scale(scheme='magma'))</code></p><p><code>)</code></p><p> </p><p><code>points = base.mark_circle().encode(</code></p><p><code>    opacity=alt.value(0),</code></p><p><code>    tooltip=[</code></p><p><code>        alt.Tooltip('Year:O', title='Year'),</code></p><p><code>        alt.Tooltip('DOY:O', title='DOY'),</code></p><p><code>        alt.Tooltip('NDVI:Q', title='NDVI')</code></p><p><code>    ]</code></p><p><code>).add_selection(</code></p><p><code>    highlight</code></p><p><code>)</code></p><p> </p><p><code>lines = base.mark_line().encode(</code></p><p><code>    size=alt.condition(~highlight, alt.value(1), alt.value(3))</code></p><p><code>)</code></p><p> </p><p><code>(points + lines).properties(width=600, height=300).interactive()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/earthengine-community/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='72'/>

Reply:
<p>Great idea! Done.</p>